tests/test_dcos_e2e/docker_utils/test_loopback.py

```diff
@@ -0,0 +1,69 @@
+from typing import Tuple
+
+import docker
+import pytest
+
+from dcos_e2e.docker_utils import DockerLoopbackVolume
+
+# We ignore this error because it conflicts with `pytest` standard usage.
+# pylint: disable=redefined-outer-name
+
+
+@pytest.fixture
+def loopback_sidecar() -> Tuple[docker.models.containers.Container, str]:
+    return DockerLoopbackVolume.create(size=1)
+
+
+class TestDockerLoopbackVolume:
+    """
+    Tests for setting device mapping on master or agent Docker containers.
+    """
+
+    def test_loopback(self) -> None:
+        """
+        An instance of `DockerLoopbackVolume` provides a block device.
+        """
+        labels = {'foo': 'bar'}
+
+        with DockerLoopbackVolume(size=1, labels=labels) as device:
+            cmd = 'lsblk {}'.format(device.path)
+            exit_code, output = device.container.exec_run(cmd=cmd)
+            assert exit_code == 0, cmd + ': ' + output.decode()
+
+            for key, value in labels.items():
+                assert device.container.labels[key] == value
+
+    def test_create(self) -> None:
+        """
+        Calling `DockerLoopbackVolume.create` creates a sidecar container.
+        """
+        labels = {'foo': 'bar'}
+
+        container, path = DockerLoopbackVolume.create(
```

Comment:
What do you think about:

```
loopback_volume = DockerLoopbackVolume(size=1, labels=labels)
container = loopback_volume.container
path = loopback_volume.path
```

?

Reply:
I'm explictly testing the static `DockerLoopbackVolume.create` method here.